23&22/json.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+print("Creating JSON output on food.js...")
+
+conn = sqlite3.connect('foodfacts.sqlite')
+cur = conn.cursor()
+
+cur.execute('''SELECT data.foodId, foods.name, data.energy, data.fat, data.carbohydrates, data.fiber, data.proteins, data.salt, foods.url 
+    FROM foods JOIN data ON foods.id = data.foodId
+    ''')
+
+def check(data):
+    if data == "?" :
+        return -9999
```

Comment:
Have you tried to return the string "?" if it's not a number, if it's a number return the number?

Reply:
you mean 
def check(data)
   if data == "?":
      return "?"
   else:
      return data